dask/delayed.py

```diff
@@ -721,3 +739,9 @@ def __call__(self, *args, **kwargs):
 def single_key(seq):
     """Pick out the only element of this list, a list of keys"""
     return seq[0]
+
+
+# TODO: Need to reorganize modules to avoid circular import, and allow
+#       import of this at the top. As of now this import needs to come
+#       after the definition of `def delayed` and `class Delayed`.
+from .graph_manipulation import bind as _bind
```

Comment:
This is obviously not ideal. Didn't want to spend to reorganizing if there is no appetite for this change. If there is, can look into this